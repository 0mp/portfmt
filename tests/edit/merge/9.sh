echo "MASTER_SITES=" | ${PORTEDIT} merge 9.in | diff -L 9.expected -L 9.actual -u 9.expected -
