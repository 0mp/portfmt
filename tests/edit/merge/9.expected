PORTNAME=	cloak
MASTER_SITES=
